sentry.py

```diff
@@ -0,0 +1,101 @@
+import os
+import uuid
+import hashlib
+import psutil
+import sentry_sdk
+import socket
+import time
+
+sentry_sdk.init(
+    # https://docs.sentry.io/concepts/key-terms/dsn-explainer/
+    dsn="https://3a80dc66ff544e5000cb4c50751f0eca@o4508651588485120.ingest.us.sentry.io/4508651601526784",
```

Comment:
Okay, I need to rotate this one, but Sentry tells that it's fine to ship them to users.

P. S. This file should be removed before PR is merged.